src/libraries/System.Security.Cryptography/tests/System.Security.Cryptography.Tests.csproj

```diff
@@ -4,11 +4,20 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(CommonTestPath)System\Security\Cryptography\PlatformSupport.cs"
-             Link="CommonTest\System\Security\Cryptography\PlatformSupport.cs" />
-    <Compile Include="$(CommonTestPath)System\IO\PositionValueStream.cs"
-             Link="CommonTest\System\IO\PositionValueStream.cs" />
+    <Compile Include="$(CommonTestPath)System\Security\Cryptography\ByteUtils.cs" Link="CommonTest\System\Security\Cryptography\ByteUtils.cs" />
```

Comment:
We use `Link` on a new line in all Compile items across the repo.

Reply:
Well, "all" clearly isn't true, since when I merged them in I had some flat ones (notably the Stream-based files) and some split ones.  Apparently I chose the wrong way to normalize :wink:

Reply:
Hah, yeah we definitely miss some, but it seems like using a new line is the preferred way?